playbook/app/pb_kits/playbook/pb_body/docs/_body_styled.md

```diff
@@ -0,0 +1,3 @@
+Playbook styles the `b`, `strong` and `a` tags within the body kit.
+* `b` and `strong` tag styles: `font-weight: 700` and `letter-spacing: -0.03`
+* `a` tag styles:  `color: $primary` and `color: $text_lt_default` on hover.
```

Comment:
Can we just say it's styled like the title size 4? We don't do this breakdown of the styles in any other docs. I don't suggest doing it here. What if we change the letter spacing styles...then we have to remember to update the docs?